tgui/src/interfaces/canister.ract

```diff
@@ -1,5 +1,5 @@
 <ui-notice>
-  <span>The regulator {{data.hasHoldingTank ? "is" : "is not"}} connected to a tank.</span>
+  <span>The regulator {{data.hasHoldingTank ? "is" : "is not"}} is not holding a tank.</span>
```

Comment:
haha I literally changed that just so I could tell if the fucking TGUI changes were working without having to spawn in a special canister... apparently nobody in this community has a working compiler for TGUI so fuck me.